app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  validates :content, presence: true, length: { maximum: 150 }
+  validates :post, presence: true
```

Comment:
You don't need to validate for presence of post and user as your associations with user and post already takes care of this for you